assets/sass/protocol/includes/themes/_firefox.scss

```diff
@@ -35,51 +35,101 @@
 
 %firefox-type-scale {
     // typescale
-    --title-2xl-size: 4rem;
-    --title-2xl-line-height: 1.125;
-    --title-xl-size: 3.5rem;
-    --title-xl-line-height: 1.143;
-    --title-lg-size: 3rem;
-    --title-lg-line-height: 1.166;
-    --title-md-size: 2.5rem;
-    --title-md-line-height: 1.1;
-    --title-sm-size: 2rem;
-    --title-sm-line-height: 1.125;
-    --title-xs-size: 1.5rem;
-    --title-xs-line-height: 1.166;
-    --title-2xs-size: 1.25rem;
-    --title-2xs-line-height: 1.2;
+    --title-2xl-size: 3.5rem;
```

Comment:
I've been toying with the idea of variable names that make it clear that they morph with media queries. Alternative name ideas:

- `--scale-title-2xl-size: 3.5rem;`(colours could use `theme`, nice because it gives a clue of the variable value)
- `--let-title-2xl-size: 3.5rem;` (javascript reference: const vs let, mostly nice because its short, we should not have any const CSS vars, they should be Sass vars)
- `--find-title-2xl-size: 3.5rem;` (nice because all dynamic variables can use the same prefix)

Reply:
Minor preferences:
- scale/theme prefixes on CSS custom properties look good to me, I like the extra indication of change based on context
- `var()` syntax doesn't bother me but happy to save a few characters with alias to Sass variables